src/MMU.cpp

```diff
@@ -259,3 +354,8 @@ void MMU::setBit(uint16_t addr, uint8_t pos, uint8_t value) {
 	data |= (value << pos);
 	memory[addr] = data;
 }
+
+void MMU::setIF(uint8_t pos, uint8_t value) {
```

Comment:
With the addition of setIF, setBit is no longer used. Going to leave it in for now since it may be useful later on.